presto-main-base/src/main/java/com/facebook/presto/execution/SafeEventLoopGroup.java

```diff
@@ -92,7 +92,7 @@ public void execute(Runnable task, Consumer<Throwable> failureHandler, Scheduler
                     task.run();
                 }
                 catch (Throwable t) {
-                    log.error("Error executing task on event loop", t);
+                    log.error("Error executing method %s on event loop: %s", methodSignature, t);
```

Comment:
why two %s ?
`log.error(t, "Error executing method %s on event loop", methodSignature);`

this should suffice right ?